pkg/signatures/factory.go

```diff
@@ -50,27 +50,24 @@ func VerifyAgainstSignatureIntegration(ctx context.Context, integration *storage
 	verifiers := createVerifiersFromIntegration(integration)
 	var results []*storage.ImageSignatureVerificationResult
 	for _, verifier := range verifiers {
-		res, err := verifier.VerifySignature(ctx, image)
+		res, verifiedImageReferences, err := verifier.VerifySignature(ctx, image)
+
+		verificationResult := &storage.ImageSignatureVerificationResult{
+			VerificationTime: protoconv.ConvertTimeToTimestamp(time.Now()),
+			VerifierId:       integration.GetId(),
+			Status:           res,
+		}
 		// We do not currently support specifying which specific method within an image signature integration should
 		// be successful. Hence, short-circuit on the first successfully verified signature within an image signature
 		// integration.
 		if res == storage.ImageSignatureVerificationResult_VERIFIED {
+			verificationResult.VerifiedImageReferences = verifiedImageReferences
```

Comment:
I initially wanted to be explicit and only add the verified image references in case the signature was also verified. 
But since we don't make this differentiation anymore, I'll just add it unconditionally.